gemrb/GUIScripts/GUICG13.py

```diff
@@ -159,13 +159,13 @@ def GetColor():
 		Button.OnPress (DonePress)
 
 	ColorPicker.GetControl(0).MakeEscape ()
-	ColorPicker.ShowModal (MODAL_SHADOW_NONE)
+	ColorPicker.ShowModal (MODAL_SHADOW_GRAY)
 	return
 
 def HairPress():
 	global ColorIndex, PickedColor
 
-	ColorWindow.SetVisible(False)
+	ColorWindow.SetVisible(True)
```

Comment:
What's the rationale here? These SetVisible changes look bogus.

Reply:
Ah sorry, I forgot it in the description. The colorwindow is not supposed to disappear when you select a color:

Original:
![wine](https://github.com/gemrb/gemrb/assets/155195419/a2911a52-1419-439d-b885-f03d5220bd61)

GemRB:
![gemrb](https://github.com/gemrb/gemrb/assets/155195419/bdb8e6de-2438-4740-8597-db3a5284a54d)

Thanks for making me checking it again. Now I discovered two more bugs with the color window, but that's for another PR (Major and Minor button press actions are swapped, and ESC-ing out of the color picking window changes the color to some default color).

Reply:
It might be from bg2 then, but we might get away without this there as well, since the two windows look like they overlap perfectly (once the placement bug is fixed). But that would mean we can remove all these calls, since there's nothing setting them to invisible.

Reply:
Of course! Thanks, I removed them.